app/uk/gov/hmrc/tradergoodsprofilesdatastore/models/response/Pagination.scala

```diff
@@ -41,10 +41,13 @@ object Pagination {
     val page                 = pageOpt.getOrElse(localStartingPage)
     val mod                  = totalRecords % size
     val totalRecordsMinusMod = totalRecords - mod
-    val totalPages           = ((totalRecordsMinusMod / size) + 1).toInt
+    val totalPages           = (if (mod == 0 && totalRecords != 0) {
+                        totalRecordsMinusMod / size
+                      } else {
+                        (totalRecordsMinusMod / size) + 1
+                      }).toInt
```

Comment:
Indentation looks nice this way :D 
`val totalPages           = {
  if (mod == 0 && totalRecords != 0) {
    totalRecordsMinusMod / size
  } else {
    (totalRecordsMinusMod / size) + 1
  }
}.toInt`